src/main/scala/mesosphere/marathon/state/Residency.scala

```diff
@@ -0,0 +1,16 @@
+package mesosphere.marathon.state
+
+import com.wix.accord.dsl._
+import mesosphere.marathon.Protos.ResidencyDefinition.TaskLostBehavior
+
+case class Residency(relaunchEscalationTimeoutSeconds: Long, taskLostBehavior: TaskLostBehavior)
```

Comment:
Minor: specifying a relaunchEscalationTimeoutSeconds only makes sense for a certain behavior.